samples/excel/20-data-types/data-types-error-values.yaml

```diff
@@ -29,13 +30,35 @@ script:
           });
         }
 
+        async function changeBusyError() {
+          // This function checks if a range contains a #BUSY! error, and then updates the range if it contains the error.
+          await Excel.run(async (context) => {
+            // Retrieve the Sample worksheet and cell A1 on that sheet.
+            const sheet = context.workbook.worksheets.getItemOrNullObject("Sample");
+            const range = sheet.getRange("A1");
+
+            // Load the `valuesAsJson` property for comparison.
+            range.load("valuesAsJson");
+            await context.sync();
+
+            // Check if the range contains a #BUSY! error.
+            if (range.valuesAsJson[0][0]["errorType"] == Excel.ErrorCellValueType.busy) {
```

Comment:
This might be overkill for the sample, but consider upgrading this to change all busy cells in the used range.

Reply:
Great idea! Thanks. Will update the sample to use getUsedRange.